Playmate/Utilities/Constants.m

```diff
@@ -23,15 +23,24 @@ + (NSString *)decathalonSportsListString {
     return @"https://sports.api.decathlon.com/sports";
 }
 
++ (NSString *)decathalonOneSportString {
+    return @"https://sports.api.decathlon.com/sports/:";
+}
+
+// For Parse
++ (PlayerConnection *)getPlayerConnectionForUser:(PFUser *)user {
+    return [user[@"playerConnection"][0] fetchIfNeeded];
```

Comment:
Hmm I'm not quite sure if I can add a method in user since its a pre defined Parse PFUser object. I'll think of a better way, but for now i'll move it to Helpers file instead of constants